packages/material-ui-lab/src/Slider/Slider.js

```diff
@@ -119,6 +119,15 @@ export const styles = theme => {
         height: 17,
       },
     },
+    thumbTransparent: {
+      backgroundColor: 'transparent',
+    },
+    thumbIcon: {
```

Comment:
Please add a block comment to document this `className`.

Reply:
I have added a small documentation to it.